testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeAlterTableCompatibility.java

```diff
@@ -182,7 +185,7 @@ public void testCommentOnTable()
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_DATABRICKS_104, DELTA_LAKE_DATABRICKS_113, DELTA_LAKE_DATABRICKS_122, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
```

Comment:
The condition of when we should run all is unclear to me.

1. The purpose of this `testCommentOnTableUnsupportedWriterVersion` is ensuring the failure of new writer version, so I thought running on the newest LTS version is sufficient. 
2. When we see this entire class, I wonder why it covers only COMMENT ON COLUMN statement.

Reply:
Removed the testing from TestDeltaLakeAlterTableCompatibility.

Reply:
> The condition of when we should run all is unclear to me.

+1 

We are currently running a reduced set of tests < 10 minutes to see whether there is anything obvious failing.
And also to make sure that the product test environment for the Databricks version 10.x , 11.x, 12.x doesn't rot.